.storybook/main.ts

```diff
@@ -10,7 +10,8 @@ const config: StorybookConfig = {
 		{
 			name: "@storybook/addon-styling",
 			options: {}
-		}
+		},
+		"@storybook/addon-mdx-gfm"
```

Comment:
In the [docs](https://storybook.js.org/addons/@storybook/addon-mdx-gfm) it says that this package might be removed in the future.

According to what they say, there's another option to get github flavored markdown with a custom remark plugin [1](https://storybook.js.org/docs/writing-docs/mdx#lack-of-github-flavored-markdown-gfm)

Reply:
You're completely right! Let's delete it.